lua/wikis/ageofempires/Infobox/Person/Player/Custom.lua

```diff
@@ -262,9 +254,6 @@ end
 function CustomPlayer:adjustLPDB(lpdbData, args, personType)
 	lpdbData.region = Region.name{country = args.country}
 
-	lpdbData.extradata.role = args.roleList[1]
-	lpdbData.extradata.role2 = args.roleList[2]
-	lpdbData.extradata.roles = mw.text.listToText(args.roleList)
```

Comment:
This breaks storage; the new commons implementation is not equivalent right now.

Reply:
Seems fine, i think we can even remove the `args.roleList` array used here and use `self.roles` instead.

I don't exactly remember why `mw.text.listToText` was used here, but if that was necessary can readd it in the downstream usage.